package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "eslint .",
     "lintfix": "eslint . --fix",
     "coverage": "npm run nyc",
-    "nyc": "nyc npm test"
+    "nyc": "nyc npm test",
+    "release-version": "bin/changelog"
```

Comment:
If you add it to the `bin` object in the package.json, it is installed as a first class command. See https://github.com/littlstar/cfs-cli/blob/master/package.json#L15

Reply:
Ah, good point.  Will move this to a /scripts directory.